src/Token.php

```diff
@@ -68,27 +68,37 @@ public function clear()
         $this->driver->set('timeToken', $this->time);
     }
 
-    /**
-     * @param      $keys
-     * @param null $default
-     */
     public function getMultiple($keys, $default = null)
     {
-    }
+        $cache = [];
+        foreach ($keys as $key) {
+            $cache[$key] = $this->get($key, $default);
+        }
 
-    /**
-     * @param      $values
-     * @param null $ttl
-     */
-    public function setMultiple($values, $ttl = null)
+        return $cache;
+    }
+	/**
+	/*Format like: $arrays = [
```

Comment:
The comment is incorrect.

Reply:
you can correct that bro.

Reply:
Yes I can, but in code review it's not the point to fix code for someone else but for give feedback.

Reply:
i am unable to edit this commit again because i had deleted the forked repo.